raster-cli/public/ImageFactory.hpp

```diff
@@ -0,0 +1,38 @@
+#pragma once
```

Comment:
This is not the correct design pattern for a factory. So, submit it in that way, but I will make a change which changes it.

Reply:
This is the [factory method design pattern](https://sourcemaking.com/design_patterns/factory_method). I decided to implement it like this because I needed a simple testing method. I decided to push it with this pull-request as a method to use the Image class until something better is implemented. I support the proper implementation of the factory design pattern.